src/cpp/server/db/20210916132211194382021_Prairie-Trillium_AlterTables.postgresql

```diff
@@ -0,0 +1,88 @@
+/* Stores users of RStudio Server */
+CREATE TABLE licensed_users(
+
+   /* The username associated with the user */
+   user_name text NOT NULL,
+
+   /* Always false column, to keep the order the same as in Workbench */
+   locked boolean NOT NULL DEFAULT 0,
+
+   /* The date time when the user last signed in */
+   last_sign_in text NOT NULL,
+
+   /* Always false column, to keep the order the same as in Workbench */
+   is_admin boolean NOT NULL DEFAULT 0,
+
+   /* The Posix/Operating-System-Level ID of the user */
+   user_id integer NOT NULL DEFAULT -1,
+
+   /* Auto incrementing ID for each user for internal FK relationships. */
+   id integer NOT NULL,
+
+   /* Add Primary Key */
+   PRIMARY KEY(id)
+
+);
+
+/* Stores metadata about actvie sessions */ 
+CREATE TABLE active_session_metadata(
+
+   /* The ID of the session which this row describes */
+   session_id text NOT NULL,
+
+   /* The ID of the row in licensed_users which corresponds to the user that owns this session */
+   user_id integer NOT NULL,
+
+   /* The type of workbench for this session */
+   workbench text NOT NULL,
+
+   /* The time at which this session was first created */
+   created text NOT NULL,
+
+   /* The time at which this session was last used */
+   last_used text NOT NULL,
+
+   /* The version of R in use, if this session's workbench type is R */
+   r_version text,
+
+   /* The label of the version of R in use, if this session's workbench type is R */
+   r_version_label text,
+
+   /* The R home of the version of R in use, if this session's workbench type is R */
+   r_version_home text,
+
+   /* The open project, if any */
+   project text,
+
+   /* The working directory of the session */
+   working_directory text,
+
+   /* The state of the session (running, executing, suspended) */
+   activity_state text NOT NULL,
+
+   /* The display label for this session */
+   label text NOT NULL,
+
+   /* The parameters with which this session was launched */
+   launch_parameters text NOT NULL,
+
+   /* Whether or not a save prompt is required */
+   save_prompt_required text NOT NULL DEFAULT 'not_required',
+
+   /* The size of the session on disk, if it is curruently suspended */
+   suspended_size_kb integer,
```

Comment:
While I was working on this, I found that the suspended size isn't written to a file on disk - it's the actual size of the session when it's suspended on disk. Since we need to read/write to that file for session suspension/resumption, I think we should always go to the ultimate source of truth for this value.

Also, IIRC, the value doesn't come from the ActiveSession object, so the current implementation plan doesn't include a way to read/write this value from the Session itself.
```suggestion
```